templates/static/cookie-consent/ds-cookie-consent.css.map

```diff
@@ -1 +0,0 @@
-{"version":3,"sourceRoot":"","sources":["../src/ds-cookie-consent.scss"],"names":[],"mappings":"AACE;EACE;;AACA;EACE;;AACA;EAFF;IAGI;;;AAKN;EACE;EACA;;AAGF;EACE;;AAGF;EACE;;;AAIJ;EACE;EACA;EACA;EACA;EACA;EACA;EACA;EACA;;AACA;EACE;EACA;EACA;EACA;;AACA;EACE;EACA;EACA;EACA;EACA;EACA;EACA;EACA;;AAEF;EACE;EACA;EACA;EACA;EACA;EACA;;AACA;EACE;EACA;EACA;EACA;EACA;;AAEF;EACE;EACA;;AACA;EACE;EACA;EACA;EACA;EACA;;AAEF;EACE;;AAIN;AAAA;AAAA;EAGE;EACA;EACA;EACA;EACA;EACA;EACA;EACA;EACA;EACA;EACA;;AACA;AAAA;AAAA;AAAA;AAAA;EAEE;EACA;EACA;;AAEF;AAAA;AAAA;EACE;;AAEF;EAvBF;AAAA;AAAA;IAwBI;IACA;IACA;;;AAGJ;EACE;EACA;EACA;EACA;;AACA;EAEE;EACA;EACA;EACA;;AAEF;EACE;;AAEF;EAfF;IAgBI;;;AAEF;EAlBF;IAmBI;;;AAGJ;EA/FF;IAgGI;;;;AAMJ;EAEI;IACE;;;;AAON;EACE;;AAEF;EACE","file":"ds-cookie-consent.css"}
```

Comment:
Technically out of scope, but I've added a line to the .gitignore so we're not commiting css maps.